LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Tomas Norre Mikkelsen
+Copyright (c) 2020 Philipp Kuhlmay
```

Comment:
The License info here and in the files doesn't match regarding naming, perhaps a LICENSE file should be enough, and then just reference it in the PHP files, I'm considering doing the same for the crawler, when I get all files and the same license. 

But here we have the chance to do it from the beginning.